.github/workflows/development.yml

```diff
@@ -57,12 +57,12 @@ jobs:
               run: echo "Database URL $DATABASE_URL; PG_USER $PG_USER; PG_PW $PG_PW; PG_DB $PG_DB"
 
             - name: '☁️ checkout repository'
-              uses: actions/checkout@v3
+              uses: actions/checkout@v4
 
             - name: '🔧 Setup Node.js'
-              uses: actions/setup-node@v3
+              uses: actions/setup-node@v4
               with:
-                  node-version: 'lts/*'
+                  node-version: '18'
```

Comment:
is there a reason for using 18 instead of 20?

Reply:
Erm no not really, gonna revert that to "lts/*"